src/curated-corpus/pages/CustomSectionsPage/CustomSectionsPage.tsx

```diff
@@ -0,0 +1,488 @@
+import React, { useEffect, useState } from 'react';
+import { DateTime } from 'luxon';
+import {
+  Section,
+  SectionStatus,
+  useGetScheduledSurfacesForUserQuery,
+  useGetSectionsWithSectionItemsLazyQuery,
+} from '../../../api/generatedTypes';
+import { DropdownOption } from '../../helpers/definitions';
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  Paper,
+  Chip,
+  TextField,
+  MenuItem,
+} from '@mui/material';
+import { HandleApiResponse } from '../../../_shared/components';
+
+interface GroupedSections {
+  scheduled: Section[];
+  live: Section[];
+  expired: Section[];
+}
+
+export const CustomSectionsPage: React.FC = (): JSX.Element => {
+  const [currentScheduledSurfaceGuid, setCurrentScheduledSurfaceGuid] =
+    useState('');
+  const [scheduledSurfaceOptions, setScheduledSurfaceOptions] = useState<
+    DropdownOption[]
+  >([]);
+  const [groupedSections, setGroupedSections] = useState<GroupedSections>({
+    scheduled: [],
+    live: [],
+    expired: [],
+  });
+
+  // Get sections with section items
+  const [
+    callGetSectionsWithSectionItemsQuery,
+    { data, loading, error, refetch },
+  ] = useGetSectionsWithSectionItemsLazyQuery({
+    fetchPolicy: 'cache-and-network',
+    notifyOnNetworkStatusChange: true,
+    variables: {
+      scheduledSurfaceGuid: currentScheduledSurfaceGuid,
+    },
+    onCompleted: () => {
+      // Query completed
+    },
+  });
+
+  // Get scheduled surfaces for user
+  useGetScheduledSurfacesForUserQuery({
+    onCompleted: (data) => {
+      const options = data.getScheduledSurfacesForUser.map(
+        (scheduledSurface) => {
+          return { code: scheduledSurface.guid, name: scheduledSurface.name };
+        },
+      );
+      if (options.length > 0) {
+        setCurrentScheduledSurfaceGuid(options[0].code);
+        setScheduledSurfaceOptions(options);
+      }
+      callGetSectionsWithSectionItemsQuery();
+    },
+  });
+
+  // Update scheduled surface
+  const updateScheduledSurface = (option: DropdownOption) => {
+    refetch && refetch({ scheduledSurfaceGuid: option.code });
+    setCurrentScheduledSurfaceGuid(option.code);
+  };
+
+  // Group sections by status
+  useEffect(() => {
+    if (data?.getSectionsWithSectionItems) {
+      const sections = data.getSectionsWithSectionItems;
+
+      // Filter only MANUAL sections (custom sections)
+      const customSections = sections.filter(
+        (section) => section.createSource === 'MANUAL',
```

Comment:
we don't need to do this here, since we added a `createSource` filter for the `getSectionsWithSectionItems` query